bolt/lib/Profile/DataAggregator.cpp

```diff
@@ -115,6 +115,12 @@ cl::opt<std::string>
                             "perf-script output in a textual format"),
                    cl::ReallyHidden, cl::init(""), cl::cat(AggregatorCategory));
 
+cl::opt<bool>
+    ReadPerfScript("perfscript",
+                   cl::desc("skip perf and read perf-script trace created by "
+                            "Linux perf tool with script command"),
+                   cl::ReallyHidden, cl::cat(AggregatorCategory));
```

Comment:
```suggestion
                   cl::Hidden, cl::cat(AggregatorCategory));
```

I believe we could make the new flag just `Hidden` instead, so it can appear with the below?

```
 perf2bolt --help-list-hidden
```

The old flag (`ReadPerfEvents`) was never really used from the CLI, only in a single unit test (PerfSpeEvents), since it was never fully implemented. We could replace it and drop the old flag, given that we hardcode the parsing buffer to the values used by the relevant unit test?